src/node-property.cpp

```diff
@@ -155,7 +155,7 @@ namespace viewer {
         "  gui.callVoidProperty(nodeName,\"%1\")");
     QPushButton* button = new QPushButton(objectName());
     button->setToolTip (toolTip.arg(objectName()));
-    connect(button, SIGNAL(clicked()), SLOT(set(void)));
```

Comment:
You are using a C++11 feature.

Reply:
Yes, and this time, it is on purpose. I will make it clear in CMake.